PullRequestCreated/PullRequestCreated.ts

```diff
@@ -43,9 +63,20 @@ class PullRequestCreated extends OctokitAction {
         if (this.payload.pull_request.requested_reviewers.length > 0) { // When PR is created directly with a reviewer, process it here. RequestReview action can be scheduled faster and PR title might not have an issue ID yet
           await this.processRequestReview(issueId, this.payload.pull_request.requested_reviewers[0]);
         }
+        return issueId;
       }
+    } else {
+      return null;
     }
-    return newTitle;
+  }
+
+  private async persistIssueId(pr: PullRequest, issueId: string): Promise<void> {
+    pr.title = this.cleanupWhitespace(`${issueId} ${pr.title}`);
+    await this.updatePullRequestTitle(pr.number, pr.title);
+  }
```

Comment:
This extraction was not necessary for this PR. It's done as a preparation for GHA-18